modules/formulize/include/formulizeConfigSync.php

```diff
@@ -0,0 +1,797 @@
+<?php
+
+include_once XOOPS_ROOT_PATH . "/modules/formulize/include/formulizeConfigSyncElementValueProcessor.php";
+
+class FormulizeConfigSync
+{
+	private $db;
+	private $configPath;
+	private $elementValueProcessor;
+	private $formHandler;
+	private $elementHandler;
+	private $changes = [];
+	private $diffLog = [];
+	private $errorLog = [];
+
+	private $configFiles = [
+		'forms' => 'forms.json'
+	];
+
+	/**
+	 * Constructor
+	 * @param string $configPath
+	 */
+	public function __construct(string $configPath)
+	{
+		$this->configPath = rtrim($configPath, '/');
+		$this->formHandler = xoops_getmodulehandler('forms', 'formulize');
+		$this->elementHandler = xoops_getmodulehandler('elements', 'formulize');
+		$this->elementValueProcessor = new FormulizeConfigSyncElementValueProcessor();
+		$this->initializeDatabase();
+	}
+
+	/**
+	 * Initialize the database connection
+	 * @return void
+	 */
+	private function initializeDatabase()
+	{
+		try {
+			// @todo we should probably be usign the $xoopsDB object
+			// But it's an older version of PDO which means that many
+			// of the newer features we're using here will need to be
+			// refactored. For now, we'll just use a new PDO object.
```

Comment:
Sounds good to me! There are other more esoteric parts of the admin side that don't use xoopsDB either.